server/monitor/src/main/java/org/apache/accumulo/monitor/rest/logs/DeadLoggerInformation.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.logs;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+/**
+ *
+ * Stores dead logger information
+ *
+ * @since 2.0.0
+ *
+ */
+public class DeadLoggerInformation {
+
+  // Variable names become JSON keys
+  @XmlAttribute
```

Comment:
In these cases, the annotation is required for those specific XML tags, here's an example
`<servers>
  <server id="aws.revelc.net:9997">
    <hostname>aws.revelc.net:9997</hostname>
  </server>
</servers>`

Here the 'id' on the 'server' tag is an attribute of that tag, if I didn't include the `@XmlAttribute` annotation then it would show up like this 

`<servers>
  <server>
   <id>aws.revelc.net:9997</id>
   <hostname>aws.revelc.net:9997</hostname>
  </server>
</servers>`